src/AspNet.Security.OAuth.Bitrix24/Bitrix24AuthenticationConstants.cs

```diff
@@ -0,0 +1,258 @@
+            public const string UfInterests = "uf_interests";
+            public const string UfSkills = "uf_skills";
+            public const string UfWebSites = "uf_web_sites";
+            public const string UfXing = "uf_xing";
+            public const string UfLinkedin = "uf_linkedin";
+            public const string UfFacebook = "uf_facebook";
+            public const string UfTwitter = "uf_twitter";
+            public const string UfSkype = "uf_skype";
+            public const string UfDistrict = "uf_district";
+            public const string UfPhoneInner = "uf_phone_inner";
+        }
+
+        internal static class UserFields
+        {
+            public const string Id = "ID";
+            public const string Active = "ACTIVE";
+            public const string Email = "EMAIL";
+            public const string Name = "NAME";
+            public const string LastName = "LAST_NAME";
+            public const string SecondName = "SECOND_NAME";
+            public const string PersonalGender = "PERSONAL_GENDER";
+            public const string PersonalProfession = "PERSONAL_PROFESSION";
+            public const string PersonalWww = "PERSONAL_WWW";
+            public const string PersonalBirthday = "PERSONAL_BIRTHDAY";
+            public const string PersonalPhoto = "PERSONAL_PHOTO";
+            public const string PersonalIcq = "PERSONAL_ICQ";
+            public const string PersonalPhone = "PERSONAL_PHONE";
+            public const string PersonalFax = "PERSONAL_FAX";
+            public const string PersonalMobile = "PERSONAL_MOBILE";
+            public const string PersonalPager = "PERSONAL_PAGER";
+            public const string PersonalStreet = "PERSONAL_STREET";
+            public const string PersonalCity = "PERSONAL_CITY";
+            public const string PersonalState = "PERSONAL_STATE";
+            public const string PersonalZip = "PERSONAL_ZIP";
+            public const string PersonalCountry = "PERSONAL_COUNTRY";
+            public const string WorkCompany = "WORK_COMPANY";
+            public const string WorkPosition = "WORK_POSITION";
+            public const string UfDepartment = "UF_DEPARTMENT";
+            public const string UfInterests = "UF_INTERESTS";
+            public const string UfSkills = "UF_SKILLS";
+            public const string UfWebSites = "UF_WEB_SITES";
+            public const string UfXing = "UF_XING";
+            public const string UfLinkedin = "UF_LINKEDIN";
+            public const string UfFacebook = "UF_FACEBOOK";
+            public const string UfTwitter = "UF_TWITTER";
+            public const string UfSkype = "UF_SKYPE";
+            public const string UfDistrict = "UF_DISTRICT";
+            public const string UfPhoneInner = "UF_PHONE_INNER";
+        }
+
+        public static class Scopes
+        {
+            /// <summary>
+            /// <a href="https://dev.1c-bitrix.ru/rest_help/bizproc/index.php" target="_blank">Бизнес-процессы</a>
```

Comment:
Please make these comments written sentences (not just a link) so they're useful in Intellisense in Visual Studio for example. Also please use English.